src/chainconsumer/chain.py

```diff
@@ -7,11 +7,13 @@
 
 There are also a few helper functions and objects in here, like the `MaxPosterior` class which
 provides the log posterior and the coordinate at which it can be found for the chain."""
+
 from __future__ import annotations
 
 import logging
 from typing import TYPE_CHECKING, Any, TypeAlias
 
+import arviz as az
```

Comment:
Hmm reading through this now I realise that that arviz is only a dependency in the `test` group, so I'd be tempted to move this import down to a conditional one where we wrap it in a `try - catch`.

The type hint should stay as `arviz.blah` because that import is only eval'd `if TYPE_CHECKING` (so shouldnt be a problem if its not there, only the type hinter will be sad)

Reply:
I move the import to the from_arviz constructor only, and didn't add the try-catch as it is very unlikely that someone using it do not have arviz